cdap-api/src/main/java/io/cdap/cdap/api/app/Application.java

```diff
@@ -46,7 +46,8 @@ default boolean isUpdateSupported() {
    * @param applicationUpdateContext Used to access methods helpful for operations like upgrading plugin version for
    * config.
```

Comment:
javadoc should include `@throws` and describe what will happen if an exception is thrown.